include/game/Coordinates.hpp

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
+#include <utility>
+
+class Coordinates {
+public:
+    struct CoordX {
+        explicit constexpr CoordX(int coordX)
+            : value(coordX) {}
+
+        constexpr bool operator==(const CoordX& other) const {
+            return value == other.value;
+        }
+
+        int value{0};
+    };
+
+    struct CoordY {
+        explicit constexpr CoordY(int coordY)
+            : value(coordY) {}
+
+        constexpr bool operator==(const CoordY& other) const {
+            return value == other.value;
+        }
+
+        int value{0};
+    };
+
+    constexpr Coordinates(const CoordX& posX, const CoordY& posY)
+        : posX_(posX), posY_(posY) {}
+
+    constexpr std::pair<CoordX, CoordY> getPosition() const {
```

Comment:
Not sure we need this method (Coordinates is position). Maybe distance or some kind of lerp will be enough.
Assuming that we will have separate types for coordinates, consider writing the operator == for each individual and for Coordinates as whole.